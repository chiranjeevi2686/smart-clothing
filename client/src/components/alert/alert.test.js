import React from 'react';
import { render } from '@testing-library/react';
import SmartAlert from './alert.component';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';

const defaultStore = {
};
const mockStore = configureStore();
describe('SmartAlert component', () => {
    it('should render SmartAlert', () => {
        const { asFragment, getByText } = render(<Provider store={mockStore(defaultStore)}>
            <SmartAlert varient='success' message='Your payment is successful'/>
        </Provider>)
        expect(asFragment()).toMatchSnapshot();
        expect(getByText('Your payment is successful')).toBeInTheDocument();
    })
})

