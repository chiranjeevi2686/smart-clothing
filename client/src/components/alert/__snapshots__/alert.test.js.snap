// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmartAlert component should render SmartAlert 1`] = `
<DocumentFragment>
  <div
    class="fade alert alert-success show"
    role="alert"
  >
    Your payment is successful
  </div>
</DocumentFragment>
`;
